tests/browser_test.py

```diff
@@ -1,20 +1,39 @@
 import unittest
 import lxml.html as lh
+from octbrowser import __version__ as ob_version
 from octbrowser.browser import Browser
 
+import threading
+import SocketServer
```

Comment:
Be careful, the SocketServer module has been removed in python3, so you must use a try...except block here too

try:
    import SocketServer
except ImportError:
    import socketserver as SocketServer